.github/workflows/run-unit-tests.yml

```diff
@@ -99,8 +99,8 @@ jobs:
               break
             else
               echo "Failed to start Colima. Attempt $i/$ATTEMPT_LIMIT."
-              if [ $i -eq $ATTEMPT_LIMIT ]; then
-                colima delete
+              if [ $i -eq $(($ATTEMPT_LIMIT - 1)) ]; then
```

Comment:
CC: @tobitege

Reply:
Shouldn't it delete after every failed attempt (except the last one), like this?

```py
if [ $i -lt $ATTEMPT_LIMIT ]; then
```

Reply:
Yes.

Reply:
Ok, I'll change that in my PR then